_component_design/buttons-iconography.md

```diff
@@ -0,0 +1,354 @@
+---
+title: Button Icons
+parent: Inputs and Controls
+layout: component
+category: Components
+usage: >
+  An icon button enables a user to perform an action or initiate a process on a Control Panel page. A button directs the user's attention to important actions.
```

Comment:
i feel like this description should communicate that this page is about iconography for buttons (rather than restating what a button is for.)

Reply:
Right, I just looked at that, too - based on the title change. That said, I came up with "Button iconography refers to the use of icons in buttons." We can include more verbiage if you want, felt unnecessary to me. Any thoughts you have most welcome.